MapboxMobileEvents/Protocols/MMEEventConfigProviding.h

```diff
@@ -61,6 +61,9 @@ NS_ASSUME_NONNULL_BEGIN
 /// Legacy User-Agent String
 @property (nonatomic, copy, readonly) NSString *legacyUserAgentString;
 
+/// Unique Identifier for the client
+@property (nonatomic, copy, readonly) NSString *clientId;
```

Comment:
This should be `configuationId`

Reply:
Could you elaborate on this?

This particular change was a port from:
https://github.com/mapbox/mapbox-events-ios/commit/48145e285c68ac9f5323a1232f3c20f7ba1aee46

In which the cid: Client id was documented in the string constant.

I was wondering if I also need to update documentation for what this stands for as reading through code, I'm not sure what it represents? The unique instance of a MME SDK in an app? Something else?

As It looks to only be used internally for:
- Vendor id if used on platform != (iOS | tvOS)